docker-compose.yml

```diff
@@ -37,6 +37,24 @@ services:
       - "-c"
       - "/app/zkevm-node run --network local --cfg /app/config.toml --components rpc"
 
+  zkevm-non-sequencer-json-rpc:
```

Comment:
`zkevm-non-sequencer-json-rpc`: relay tx to the trusted node
`zkevm-permissionless-node`: sync trusted state

probably we need to merge them and also review how we are configuring them because we have the `URL` of the `Trusted Sequencer` provided by the `SMC`